### PR TITLE
Adds package description to meta.yaml file

### PR DESCRIPTION
**Pull request by marodrig**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

This PR adds a short description of the UXarray package.

### CODE REVIEW

**Comment by michaelavs** [comment]
I think this looks good overall, but I believe you will need to bump the build number on line 12 in the file

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ source:
   sha256: e4afaf5a1383c1a3be00fd4870877a1b1b6e4f893823318146a60138cbc38b38
 
 build:
-  number: 1
+  number: 2
```
Comment (erogluorhan) [inline recipe/meta.yaml:12]: The current latest version is 2022.07.0 with the build number 0. This PR will not change the version but the build number, so its build number needs to be incremented to 1, not to 2

Comment (marodrig) [reply recipe/meta.yaml]: fixed.

**Review by erogluorhan — APPROVED** [review]
Thanks very much!

**Pull request merged**